app/src/main/res/values-pl-rPL/strings.xml

```diff
@@ -0,0 +1,142 @@
+<resources>
+    <string name="app_name">DankChat</string>
+    <string name="app_name_dank">DankChat (Dank)</string>
+    <string name="send_hint">Wyślij wiadomość</string>
+    <string name="login">Zaloguj się</string>
+    <string name="reload_emotes">Przeładuj emotki</string>
+    <string name="reconnect">Połącz ponownie</string>
+    <string name="clear_chat">Wyczyść czat</string>
+    <string name="add_dialog_title">Dodaj kanał</string>
+    <string name="dialog_ok">OK</string>
+    <string name="dialog_cancel">Anuluj</string>
+    <string name="dialog_dismiss">Zignoruj</string>
+    <string name="error_dialog_copy">Kopiuj</string>
+    <string name="error_dialog_title">Złapany błąd: %1$s</string>
```

Comment:
Inconsistency with line 120, change to "Złapany wyjątek: %1$s"

Reply:
While "wyjątek" is a bit more professional, I think "błąd" is something easier to understand for most end-users, so I will change the other expression in line 120. to "błąd". Resolved in `eb7f80b`